lib/nodejs/swagger-store.js

```diff
@@ -5,17 +5,24 @@ const crypto = require('crypto');
 const path = require('path');
 const routeMap = Object.create(null);
 const validUrl = require('valid-url');
+const parseTemplateObject = require('@rbarilani/parse-template-object');
 
 module.exports = ({hexo}) => {
 
+  let swaggerParserOptions = Object.assign({}, (hexo.config.theme_config.swagger_parser || {}));
```

Comment:
This wil break if `theme_config` is not provided. We will have to make it `required` or provide default for it.

Reply:
@bhaskarmelkani this might be true but not with ours... try removing the entire `theme_config` section in your config file and see. It should not break.

Reply:
yeah, my bad. Its getting instantiated [here](https://github.com/zalando-incubator/hexo-theme-doc/blob/master/plugins/favicon.js#L8).